Frends.Community.AWS.Tests/IntegrationTests.cs

```diff
@@ -10,6 +10,7 @@
 using Newtonsoft.Json.Linq;

```

Comment:
https://github.com/CommunityHiQ/Frends.Community.AWS/blob/6f47c8cbe079ca34923bf3be3756706193a1845f/Frends.Community.AWS.Tests/IntegrationTests.cs#L168
Please switch parameters, so expected value is the first one.
https://github.com/CommunityHiQ/Frends.Community.AWS/blob/6f47c8cbe079ca34923bf3be3756706193a1845f/Frends.Community.AWS.Tests/IntegrationTests.cs#L194
Please switch parameters, so expected value is the first one.